activiti-core/activiti-engine/src/test/java/org/activiti/engine/test/bpmn/multiinstance/MultiInstanceTest.java

```diff
@@ -16,9 +16,13 @@
 package org.activiti.engine.test.bpmn.multiinstance;
 
 import static java.util.Arrays.asList;
+import static java.util.Collections.EMPTY_LIST;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singleton;
 import static java.util.Collections.singletonMap;
+import static org.activiti.engine.impl.bpmn.behavior.MultiInstanceActivityBehavior.NUMBER_OF_ACTIVE_INSTANCES;
```

Comment:
I haven't noticed that this class has `NR_ACTIVE_INSTANCES` constant that is a duplicate of the `MultiInstanceActivityBehavior`.
Is there a strong reason that, originally, `MultiInstanceActivityBehavior` constant are protected?